index.html

```diff
@@ -21,15 +26,17 @@
     </nav>
   </header>
   <main>
+
     <section>
+      <img src="img/welcome.svg" alt="Приветствие" width="456" height="350">
```

Comment:
https://up.htmlacademy.ru/htmlcss/30/criteries#d28
```suggestion
      <img src="img/welcome.svg" alt="welcome to the gorgeous sedona" width="456" height="350">
```